### PR TITLE
Tweak code in post_event_processing

### PR DESCRIPTION
**Pull request by PoignardAzur**
Inline `wants_animation_frame`.
Request new frame for both visual and accessibility updates.

### CODE REVIEW

File: crates/masonry/src/render_root.rs [hunk 1]
```diff
@@ -532,14 +532,15 @@ impl RenderRoot {
...
-        // If we need a new paint pass, make sure winit knows it.
-        if self.wants_animation_frame() {
+        if self.root.state().request_anim {
             self.state
                 .signal_queue
                 .push_back(RenderRootSignal::RequestAnimFrame);
         }
 
-        if self.root.state().needs_paint {
+        // If we either need to be repainted of need to update accessibility info,
```
Comment (waywardmonkeys) [inline crates/masonry/src/render_root.rs:541]: ... or need ...

**Review by waywardmonkeys — CHANGES_REQUESTED** [review]
Typo in comment and it really needs a commit comment (especially since it contains an actual functionality change, not just changing the animation frame handling.

File: crates/masonry/src/render_root.rs (hunk 1 above)
Comment (DJMcNab) [reply crates/masonry/src/render_root.rs:541]: (I'm not sure that this comment actually contains any information either)

**Review by DJMcNab — COMMENTED** [review]
Agree with the review from @waywardmonkeys

**Review by DJMcNab — APPROVED** [review]
Thanks

**Pull request merged**